### PR TITLE
Added authors, and renders doculabels in HTML

### PR DESCRIPTION
**Pull request by TsienJin**
- Fixed requirements file; missing jupyterlab.
- Added main.html override template for "authors" and "updated" information.
- Added authors.py hook to add in author information to the page for rendering in main.html.
- Added css styles for authors (authors.css).
- Added doculabels.py hook to convert label tags to code blocks with colours.
- Added doculabels.css for styling of doculabel tags.
- removed doculabels.js as it is no longer required.
- Updated existing markdown files to use "authors" instead of "author" in frontmatter.

---

## Key Changes

- Added authors to the bottom of the page. This is configurable in mkdocs.yaml, and controllable by frontmatter in markdown files.
- Added in python script to render @label(*) tags in HTML on the build step.

### CODE REVIEW

**Review by Wilsven — APPROVED** [review]

File: docs/projects/genai/health-hub/webapp/backend/db/mongo/er_diagram.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+---
+updated: 18 July 2024
+authors: lyndon
```
Comment (Wilsven) [inline docs/projects/genai/health-hub/webapp/backend/db/mongo/er_diagram.md:3]: Should this be standardized? I see some named as `lyndon` and other pages named as `Lyndon Lim`.

File: docs/projects/genai/health-hub/webapp/backend/api/endpoints.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
-date: 16 July 2024
-author: Ong Tsien Jin
+updated: 16 July 2024
+authors: tjay
```
Comment (Wilsven) [inline docs/projects/genai/health-hub/webapp/backend/api/endpoints.md:3]: Should this be standardized? I see some named as `tjay` and other pages named as `Ong Tsien Jin`.

File: docs/projects/genai/health-hub/webapp/backend/db/mongo/er_diagram.md (hunk 1 above)
Comment (TsienJin) [reply docs/projects/genai/health-hub/webapp/backend/db/mongo/er_diagram.md:3]: I've added support for aliases for authors -- but I must've forgotten to change this back after testing

Comment (TsienJin) [reply docs/projects/genai/health-hub/webapp/backend/db/mongo/er_diagram.md:3]: Aliases can be configured in mkdocs.yaml > extra > authors. The authors.py hook/script will match against aliases and full names

Comment (Wilsven) [reply docs/projects/genai/health-hub/webapp/backend/db/mongo/er_diagram.md:3]: Cool. Thanks for sharing!

**Pull request merged**